setup.py

```diff
@@ -0,0 +1,96 @@
+##########################################################################
```

Comment:
why having a setup.py?

Reply:
Looks like the rebase did not work as expected, will remove it.